src/Octoshift/Models/CodeScanningAnalysis.cs

```diff
@@ -0,0 +1,34 @@
+namespace Octoshift.Models
+{
+    public class CodeScanningAnalysis
```

Comment:
So far in most of our code we were able to escape creating DTOs just by using dynamic json objects. So I guess it would be possible to do so here too and it probably applies to all models that you created.